pkg/storage_inmemory/in_memory_repository.go

```diff
@@ -0,0 +1,64 @@
+package storage_inmemory
+
+import (
+	"context"
+	"github.com/pbinitiative/zenbpm/pkg/storage"
+	"sort"
+)
+
+type InMemoryStorage struct {
+	processDefinitions []storage.ProcessDefinition
```

Comment:
I would suggest to use map[id]ProcessDefinition just to get rid of some loops and faster lookup.